lib/component-data/actions.js

```diff
@@ -436,7 +444,11 @@ export function openAddComponent(store, { currentURI, parentURI, path, pos }) {
       parentURI,
       path: path,
       components: [{ name: _.head(available) }]
-    }).then((newEl) => store.dispatch('select', newEl)));
+    }).then((newEl) => {
```

Comment:
when would `newEl` not be returned?

Reply:
updating head components, or updating the layout

Reply:
could you add a code comment here? maybe something like "automatically select the new component if its not a layout or head component"

Reply:
added